### PR TITLE
AC-248907: Updating code snippets and project files

### PR DESCRIPTION
**Pull request by tatianadavydova**
Updating code snippets and customization project files for T200

### CODE REVIEW

File: Customization/T200/CodeSnippets/Step2.1.2/RSSVDevice.cs.bak [hunk 1]
```diff
@@ -0,0 +1,116 @@
+﻿using System;
```
Comment (k-s-popova) [inline Customization/T200/CodeSnippets/Step2.1.2/RSSVDevice.cs.bak]: A bak file should not be here.

**Review by k-s-popova — CHANGES_REQUESTED** [review]

**Comment by k-s-popova** [comment]
@tatianadavydova , so there are no changes in PhoneRepairProject_Code, aren't there? (Just to make sure.)

File: Customization/T200/CodeSnippets/Step2.1.2/RSSVDevice.cs.bak (hunk 1 above)
Comment (tatianadavydova) [reply Customization/T200/CodeSnippets/Step2.1.2/RSSVDevice.cs.bak]: Removed the file and updated files in the PhoneRepairProject_Code folder

File: Customization/T200/PhoneRepairShop_Code/PhoneRepairShop_Code/RSSVRepairServiceMaint.cs [hunk 2]
```diff
@@ -34,9 +37,26 @@ public class RSSVRepairServiceMaint : PXGraph<RSSVRepairServiceMaint>
...
 
-        public SelectFrom<RSSVRepairService>.View RepairService;
-
         public PXSave<RSSVRepairService> Save;
-        public PXCancel<RSSVRepairService> Cancel;
-    }
+    public PXCancel<RSSVRepairService> Cancel;
+
+
+//   public PXFilter<MasterTable> MasterView;
```
Comment (k-s-popova) [inline Customization/T200/PhoneRepairShop_Code/PhoneRepairShop_Code/RSSVRepairServiceMaint.cs]: These commented lines should be removed.

**Review by k-s-popova — CHANGES_REQUESTED** [review]
Please update to VS 2022

File: Customization/T200/PhoneRepairShop_Code/PhoneRepairShop_Code/PhoneRepairShop_Code.csproj [hunk 3]
```diff
@@ -49,22 +49,17 @@
...
     <Reference Include="System" />
     <Reference Include="System.Core">
       <RequiredTargetFramework>3.5</RequiredTargetFramework>
     </Reference>
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="RSSVDeviceMaint.cs" />
     <Compile Include="DAC\RSSVDevice.cs" />
-    <Compile Include="Data\InputData.cs" />
```
Comment (k-s-popova) [inline Customization/T200/PhoneRepairShop_Code/PhoneRepairShop_Code/PhoneRepairShop_Code.csproj:64]: This line must be restored

File: Customization/T200/PhoneRepairShop/_project/File_Bin_PhoneRepairShop_Code_dll.xml [hunk 4]
```diff
@@ -0,0 +1 @@
+﻿<File AppRelativePath="Bin\PhoneRepairShop_Code.dll" FileID="0d1e59e0-0d1f-4e0b-818f-760417aa80c5" />
```
Comment (k-s-popova) [inline Customization/T200/PhoneRepairShop/_project/File_Bin_PhoneRepairShop_Code_dll.xml:1]: It seems that the project now should as well contain File items for the csv files from InputData.

File: Customization/T200/PhoneRepairShop_Code/PhoneRepairShop_Code.sln [hunk 5]
```diff
@@ -1,10 +1,10 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
```
Comment (k-s-popova) [inline Customization/T200/PhoneRepairShop_Code/PhoneRepairShop_Code.sln]: Visual Studio Version should be 17. You need to resave the solution in the new version.

File: Customization/T200/PhoneRepairShop_Code/PhoneRepairShop_Code.sln [hunk 6]
```diff
@@ -1,10 +1,10 @@
...
-VisualStudioVersion = 16.0.31019.35
+VisualStudioVersion = 16.0.31727.386
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{E24C65DC-7377-472B-9ABA-BC803B73C61A}") = "PhoneRepairShop", "..\..\..\", "{407B75C1-DF84-479F-BD54-F5A5D7012034}"
 	ProjectSection(WebsiteProperties) = preProject
-		TargetFrameworkMoniker = ".NETFramework,Version%3Dv4.8"
+		TargetFrameworkMoniker = ".NETFramework,Version%3Dv4.7.1"
```
Comment (k-s-popova) [inline Customization/T200/PhoneRepairShop_Code/PhoneRepairShop_Code.sln]: Acumatica ERP requires .net framework 4.8 as far as I know. This change should be reverted.

File: Customization/T200/PhoneRepairShop_Code/PhoneRepairShop_Code/PhoneRepairShop_Code.csproj [hunk 7]
```diff
@@ -49,22 +49,18 @@
     <Reference Include="PX.Objects">
       <HintPath>..\..\..\..\Bin\PX.Objects.DLL</HintPath>
     </Reference>
-    <Reference Include="PX.Web.Customization, Version=1.0.0.0, Culture=neutral, PublicKeyToken=3b136cac2f602b8e, processorArchitecture=MSIL">
```
Comment (k-s-popova) [inline Customization/T200/PhoneRepairShop_Code/PhoneRepairShop_Code/PhoneRepairShop_Code.csproj:52]: Please restore these lines. They are necessary for execution of the customization plug in from the Data folder.
Please also rebuild the project with the Data folder and this reference included. Then update the bin file in the customization project.

**Review by k-s-popova — CHANGES_REQUESTED** [review]
Please fix by comments

**Review by k-s-popova — APPROVED** [review]

**Pull request merged**